hack/update-go-workspace.sh

```diff
@@ -37,6 +37,11 @@ rm -f go.work go.work.sum
 
 # Generate the workspace.
 go work init
+(
+  echo "// This is a generated file. Do not edit directly."
+  echo
+  cat go.work
+) | sponge go.work
```

Comment:
is sponge standard across Linux / Darwin?

Reply:
ohh, good question.  I think it is across Linux.  Pesky Mac.

Reply:
Actually, failed the verify job as well with "command not found"